scripts/gotests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
```

Comment:
How about adding `set -euo pipefail`? So e.g. if `compile` at L37 fails we do not `run` binary, which may be stale.